ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+1199.0 "Scarecrow Chase 2" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1200.2 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1201.2 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1202.0 "Scarecrow Chase 3" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1203.3 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1204.3 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1205.1 "Scarecrow Chase 4" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1207.0 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1212.0 "Tenebrism" sync / 1[56]:[^:]*:Galatea Magna:7A96:/
+1219.0 "Burst 1" #sync / 1[56]:[^:]*:Galatea Magna:7A97:/
+1221.0 "Burst 2" #sync / 1[56]:[^:]*:Galatea Magna:7A97:/
+1223.0 "Burst 3" #sync / 1[56]:[^:]*:Galatea Magna:7A97:/
+1225.0 "Burst 4" #sync / 1[56]:[^:]*:Galatea Magna:7A97:/
+1227.0 "Stony Gaze x4" #sync / 1[56]:[^:]*:Galatea Magna:7A99:/
+1232.0 "Dark Harvest" sync / 1[56]:[^:]*:Galatea Magna:7A9F:/ # first time tankbuster used?
+1239.1 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1244.1 "Waxing/Waning Cycle" sync / 1[56]:[^:]*:Galatea Magna:(7A91|7F6E):/
+1245.7 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F6F:/
+1246.1 "Waxing/Waning Cycle" sync / 1[56]:[^:]*:Galatea Magna:(7A93|7F70):/
+1248.1 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1256.0 "Soul Scythe" sync / 1[56]:[^:]*:Galatea Magna:7A9A:/ window 30,30 jump 1174.3
+1264.0 "Soul Nebula" sync / 1[56]:[^:]*:Galatea Magna:7A9E:/
+1275.7 "Scarecrow Chase" sync / 1[56]:[^:]*:Galatea Magna:7A9B:/
+1275.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1276.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1277.7 "Scarecrow Chase 1" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1278.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1279.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1280.7 "Scarecrow Chase 2" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1281.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1282.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1283.7 "Scarecrow Chase 3" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1284.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1285.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1286.7 "Scarecrow Chase 4" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1288.6 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1293.6 "Tenebrism" sync / 1[56]:[^:]*:Galatea Magna:7A96:/
+
+#~~~~~~~~~~#
+# Cagnazzo #
+#~~~~~~~~~~#
+
+# -p 79A3:2015.6 799C:2102.9 7986:2176.3
+# -ii 7FB7 7FB5
+# -it "Cagnazzo"
+
+2000.0 "--sync--" sync / 00:0839::Deepspine will be sealed off/ window 2000,1
+2015.6 "Stygian Deluge" sync / 1[56]:[^:]*:Cagnazzo:79A3:/
+2025.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:798F:/
+2038.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2042.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2043.3 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7991:/
+2044.3 "Body Slam" sync / 1[56]:[^:]*:Cagnazzo:(7992|7993):/
+2046.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
```

Comment:
Maybe only the first Antediluvian should be listed and the others are syncs?

Reply:
Each one is a pair of the aoes going off.

Reply:
Ah ok, then maybe number them 1 2 3.  I thought this was the orbs appearing and not the damage.